src/Adapter/HookManager.php

```diff
@@ -45,9 +46,15 @@ class HookManager
      * @param bool $use_push Force change to be refreshed on Dashboard widgets
      * @param int $id_shop If specified, hook will be execute the shop with this ID
      *
+<<<<<<< HEAD
```

Comment:
Wrong merge I think :sweat:

Reply:
Sorry, I saw it instantly, but too late.